image_proc/src/crop_decimate.cpp

```diff
@@ -119,9 +119,9 @@ CropDecimateNode::CropDecimateNode(const rclcpp::NodeOptions & options)
   int interpolation = this->declare_parameter("interpolation", 0);
   interpolation_ = static_cast<CropDecimateModes>(interpolation);
 
-  pub_ = image_transport::create_camera_publisher(this, "image_raw");
+  pub_ = image_transport::create_camera_publisher(this, "out/image");
   sub_ = image_transport::create_camera_subscription(
-    this, "image_raw", std::bind(
+    this, "in/image", std::bind(
```

Comment:
In order to maintain at least some of the existing naming, can you please change these to `out/image_raw` and `in/image_raw`?

Reply:
done